From bf593ca79df4db4bc5336ddc9c1757e726802ab9 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 22:57:44 +0800
Subject: [PATCH 35/58] Revert "drm/bridge: cdns-dsi: Make sure to to create
 proper aliases for dt"

This reverts commit 06a0001366ac0d31f3d5ae61b611173d26f22be8.
---
 drivers/gpu/drm/bridge/cdns-dsi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/cdns-dsi.c b/drivers/gpu/drm/bridge/cdns-dsi.c
index 0ced08d81d7a2..b31281f76117c 100644
--- a/drivers/gpu/drm/bridge/cdns-dsi.c
+++ b/drivers/gpu/drm/bridge/cdns-dsi.c
@@ -1286,7 +1286,6 @@ static const struct of_device_id cdns_dsi_of_match[] = {
 	{ .compatible = "cdns,dsi" },
 	{ },
 };
-MODULE_DEVICE_TABLE(of, cdns_dsi_of_match);
 
 static struct platform_driver cdns_dsi_platform_driver = {
 	.probe  = cdns_dsi_drm_probe,
-- 
2.27.0

